modules/nf-core/genmap/index/main.nf

```diff
@@ -1,29 +1,44 @@
 process GENMAP_INDEX {
-    tag "$fasta"
-    label 'process_high'
+    tag "$meta.id"
+    label 'process_low'
```

Comment:
The index process will take lots of resources like any other mapping tool. I suggest to put it as `process_high`

Reply:
Ah okay, I assumed this because there is no way to set how many resources the tool uses